core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/whitelist/ValidationWhiteListCreateRequest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.validation.whitelist;
+
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import com.b2international.snowowl.core.ApplicationContext;
+import com.b2international.snowowl.core.ComponentIdentifier;
+import com.b2international.snowowl.core.ServiceProvider;
+import com.b2international.snowowl.core.events.Request;
+import com.b2international.snowowl.core.internal.validation.ValidationRepository;
+import com.b2international.snowowl.datastore.remotejobs.RemoteJobNotification;
+import com.b2international.snowowl.eventbus.IEventBus;
+
+/**
+ * @since 6.1
+ */
+final class ValidationWhiteListCreateRequest implements Request<ServiceProvider, String> {
+
+	private String id;
```

Comment:
Auto-generate the identifier internally in this request instead of letting clients specify any value. 
Also, it is an API best practice to validate request parameters. In this case, I would check whether the specified ID value is valid or not and it is not already taken by another whitelist entry.